src/Management/src/EndpointBase/Health/HealthConverterV3.cs

```diff
@@ -38,6 +40,12 @@ public override void Write(Utf8JsonWriter writer, HealthEndpointResponse value,
 
                 writer.WriteEndObject();
             }
+
+            if (health.Groups != null && health.Groups.Count() > 0)
```

Comment:
```suggestion
            if (health.Groups?.Any() == true)
```
Typed in the browser, might want to double-check I got it right before commiting... `.Any` being the main change.  The addition of this property seems like something that should have a test or an assertion too